### PR TITLE
Codecheck bug fix.

### PR DESCRIPTION
**Pull request by hanbingleixue**
【Fix issue】#1365

[Modification content] Modify the codecheck problem
Plugin| Codecheck
-- | --
sermant-monitor| Space indentation problem
sermant-monitor| Numeric type missing suffix problem
sermant-mq-consume-deny| Missing blank lines in comments and code
sermant-router| uses custom compareTo but does not override the equals method
sermant-router| Map traversal uses keySet iterator instead of entrySet iterator
sermant-router| Public or protected method is missing annotation
sermant-router| No handling is done after catching the exception
sermant-router| Space indentation problem
sermant-router| The method is missing annotations for parameters, exceptions, or return values.
sermant-router| There should be no empty methods in abstract classes.
sermant-service-registry| Name format problem
sermant-service-registry| The reflected call should be executed in the doPrivileged method
sermant-service-registry| The current class does not override the equals method in the super class
sermant-service-registry| The current class implements the same interface as the super class
sermant-service-registry| Public or protected method is missing annotation
sermant-service-registry| There should be no empty methods in abstract methods.

[Use case description] 1. There is no need to modify the use case and there is no function modification.

[Self-test situation] 1. Passed static inspection

[Scope of influence] 1. No impact on users’ use

### CODE REVIEW

**Review by daizhenyu — DISMISSED** [review]

**Review by lilai23 — DISMISSED** [review]

File: sermant-plugins/sermant-router/router-common/src/main/java/com/huaweicloud/sermant/router/common/utils/ReflectUtils.java [hunk 1]
```diff
@@ -115,8 +115,8 @@ private static Optional<Object> invoke(Class<?> invokeClass, Object obj, String
                 return Optional.of(getAccessibleObject(invokeClass.getMethod(name, parameterClass)));
             } catch (NoSuchMethodException ignored) {
                 // 因版本的原因，有可能会找不到方法，所以可以忽略这些错误
+                return Optional.empty();
```
Comment (luanwenfei-venus) [inline sermant-plugins/sermant-router/router-common/src/main/java/com/huaweicloud/sermant/router/common/utils/ReflectUtils.java]: 这里加一个warn的日志是不是也可以

File: sermant-plugins/sermant-router/router-common/src/main/java/com/huaweicloud/sermant/router/common/utils/ReflectUtils.java [hunk 2]
```diff
@@ -126,6 +126,7 @@ private static Optional<Object> invoke(Class<?> invokeClass, Object obj, String
                 return Optional.ofNullable(method.get().invoke(obj, parameter));
             } catch (IllegalAccessException | InvocationTargetException ignored) {
                 // 因版本的原因，有可能会找不到方法，所以可以忽略这些错误
+                return Optional.empty();
```
Comment (luanwenfei-venus) [inline sermant-plugins/sermant-router/router-common/src/main/java/com/huaweicloud/sermant/router/common/utils/ReflectUtils.java]: 同上

**Review by luanwenfei-venus — DISMISSED** [review]

File: sermant-plugins/sermant-router/router-common/src/main/java/com/huaweicloud/sermant/router/common/utils/ReflectUtils.java [hunk 3]
```diff
@@ -115,8 +115,8 @@ private static Optional<Object> invoke(Class<?> invokeClass, Object obj, String
                 return Optional.of(getAccessibleObject(invokeClass.getMethod(name, parameterClass)));
             } catch (NoSuchMethodException ignored) {
```
Comment (luanwenfei-venus) [inline sermant-plugins/sermant-router/router-common/src/main/java/com/huaweicloud/sermant/router/common/utils/ReflectUtils.java]: ignore都修改一下

File: sermant-plugins/sermant-router/router-common/src/main/java/com/huaweicloud/sermant/router/common/utils/ReflectUtils.java (hunk 1 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/router-common/src/main/java/com/huaweicloud/sermant/router/common/utils/ReflectUtils.java]: 已修复

File: sermant-plugins/sermant-router/router-common/src/main/java/com/huaweicloud/sermant/router/common/utils/ReflectUtils.java (hunk 2 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/router-common/src/main/java/com/huaweicloud/sermant/router/common/utils/ReflectUtils.java]: 已修复

File: sermant-plugins/sermant-router/router-common/src/main/java/com/huaweicloud/sermant/router/common/utils/ReflectUtils.java (hunk 3 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/router-common/src/main/java/com/huaweicloud/sermant/router/common/utils/ReflectUtils.java]: 已修复

**Review by daizhenyu — APPROVED** [review]

**Review by lilai23 — APPROVED** [review]

**Review by luanwenfei-venus — APPROVED** [review]

**Review by luanwenfei-venus — APPROVED** [review]

**Pull request merged**